cmd/clusterawsadm/api/bootstrap/v1alpha1/types.go

```diff
@@ -96,6 +96,8 @@ type EKSConfig struct {
 	// AllowIAMRoleCreation controls whether the EKS controllers have permissions for creating IAM
 	// roles per cluster
 	AllowIAMRoleCreation bool `json:"iamRoleCreation,omitempty"`
+	// EnableNodesAndWorkloadsPolicy controls the creation of the policy to view EKS nodes and workloads.
+	EnableNodesAndWorkloadsPolicy bool `json:"enableNodesAndWorkloadsPolicy,omitempty"`
```

Comment:
Do you think we should change the name of this config option? It's quite long, and it doesn't match our usage of `EKS Console` in the code. Perhaps `EnableUserEKSConsolePolicy` or something similar. wdyt?

Reply:
Thanks, SGTM!